spec/features/search/new_flow/across_england/across_england_further_education_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.feature 'Searching across England' do
+  include StubbedRequests::SubjectAreas
+  include StubbedRequests::Courses
+  include StubbedRequests::Subjects
+
+  before do
+    stub_subjects
+    stub_courses_request
+    stub_subject_areas
+  end
+
+  scenario 'Candidate applies qualifications filters on results page' do
+    given_that_the_new_search_flow_feature_flag_is_enabled
+    when_i_visit_the_start_page
+    and_i_select_the_across_england_radio_button
+    and_i_click_continue
+    then_i_should_see_the_age_groups_form
+    and_the_correct_age_group_form_page_url_and_query_params_are_present
+
+    when_i_click_back
+    then_i_should_see_root_path
+    and_location_radio_selected
+
+    when_i_click_continue
+    then_i_should_see_the_age_groups_form
+
+    when_i_click_continue
+    i_should_see_an_age_group_validation_error
+
+    when_i_select_the_further_education_radio_button
+    and_i_click_continue
+    then_i_should_see_the_results_page
+  end
+
+  def given_that_the_new_search_flow_feature_flag_is_enabled
+    allow(FeatureFlag).to receive(:active?).and_call_original
+    allow(FeatureFlag).to receive(:active?).with(:new_search_flow).and_return(true)
+  end
+
+  def when_i_visit_the_start_page
+    visit root_path
+  end
+
+  def and_i_select_the_across_england_radio_button
+    choose 'Across England'
+  end
+
+  def and_i_click_continue
+    click_button 'Continue'
+  end
+
+  def when_i_click_back
+    click_link 'Back'
+  end
+
+  def when_i_click_continue
+    and_i_click_continue
+  end
+
+  def then_i_should_see_root_path
+    expect(page).to have_content('Find courses by location or by training provider')
+  end
+
+  def and_location_radio_selected
+    expect(find_field('Across England')).to be_checked
+  end
+
+  def then_i_should_see_the_age_groups_form
+    expect(page).to have_content(I18n.t('age_groups.title'))
+  end
+
+  def and_the_correct_age_group_form_page_url_and_query_params_are_present
+    URI(current_url).then do |uri|
+      expect(uri.path).to eq('/age-groups')
+      expect(uri.query).to eq('l=2')
+    end
+  end
+
+  def i_should_see_an_age_group_validation_error
+    expect(page).to have_content(I18n.t('activemodel.errors.models.search/age_groups_form.attributes.age_group.blank'))
+  end
+
+  def when_i_select_the_further_education_radio_button
+    choose 'Further education'
+  end
+
+  def then_i_should_see_the_results_page
+    expect(page).to have_content('courses found')
```

Comment:
Perhaps we may want to tighten up this assertion a little 

We could instead assert the url  and query params?

```
expect(page).to have_current_path( ... )
```

https://stackoverflow.com/questions/5228371/how-to-get-current-path-with-query-string-using-capybara